streaming/src/main/scala/io/finch/streaming/package.scala

```diff
@@ -0,0 +1,49 @@
+package io.finch
+
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.Endpoint.Result
+import io.finch.items.RequestItem
+import io.iteratee.{Enumeratee, Enumerator}
+
+/**
+  * Streaming module
+  */
+package object streaming {

```

Comment:
Looks like we can just extend this: https://github.com/travisbrown/iteratee/blob/master/twitter/src/main/scala/io/iteratee/twitter/FutureModule.scala#L9